src/components/CInput/index.tsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import Image from 'next/image';
+
+import useCustomID from '../../hooks/useCustomId';
+
+interface CInputProps {
+  icon?: string;
+  labelDetail?: string;
+  labelText?: string;
+  placeholderText?: string;
+  className?: string;
+  onChange?: (event: React.ChangeEvent<HTMLInputElement>) => void;
+  title?: string;
+}
+
+const CInput = ({
+  icon,
+  labelDetail,
+  placeholderText,
+  labelText,
+  className,
+  title,
+  onChange,
+  ...props
+}: CInputProps) => {
+  const id = useCustomID('Cinput');
+
+  return (
+    <div className={className}>
+      <div className="flex items-start">
+        <label
+          htmlFor={id}
+          className="text-slate-900 text-lg font-normal leading-[18.78px] mb-[8px]"
+          {...props}
+        >
+          {labelText}
+        </label>
+        {labelDetail && (
+          <Image src={labelDetail} alt="search" className="ml-2" title={title} />
+        )}
+      </div>
```

Comment:
Put this into a new component called CLabel
CLabel takes `labelText`, `details`, and `htmlFor`

Reply:
Remove `...props` from label